package.json

```diff
@@ -38,7 +38,7 @@
     "@azure/cosmos": "3.10.2",
     "@babel/core": "7.6.2",
     "@types/lru-cache": "5.1.0",
-    "@types/node": "12.7.9",
+    "@types/node": "^16.0.0",
```

Comment:
Should we also add `engines` into this `package.json`, just to make it clear what versions of node are supported?

Reply:
Tests currently running on `10.x` and `12.x` just fine.
Since this is optional it is fully backwards compatible with older Node.js versions.

So I want to avoid introducing a breaking change & major version increase 😄

Reply:
I mean, both are well past EOL 🤔. Do we know if we are using this anywhere on such old node versions?

Reply:
I agree about EOL and we don't use it internally, but this is a public package 🙈